po/SConscript

```diff
@@ -48,22 +72,43 @@ if "pot-update" in COMMAND_LINE_TARGETS:
         if domain in po4a_domains:
             continue
 
-        with open(join(domain, "POTFILES.in"), "w") as potfiles:
-            potfiles.writelines([line + "\n" for line in sorted(domain_sources[domain[7:]])])
-        sources = [Dir("#").File(x) for x in domain_sources[domain[7:]]]
-        if sources:
-            source_pot = env.Command(
+        # cpp
+        with open(join(domain, "POTFILES_CPP.in"), "w") as potfiles:
+            potfiles.writelines([line + "\n" for line in sorted(domain_sources_cpp[domain[7:]])])
+        sources_cpp = [Dir("#").File(x) for x in domain_sources_cpp[domain[7:]]]
+        if sources_cpp:
+            source_cpp_pot = env.Command(
                 join(domain, domain + ".cpp.pot"),
-                sources,
+                sources_cpp,
                 """xgettext --force-po --default-domain=%s --directory=. --add-comments=TRANSLATORS: \
                 --from-code=UTF-8 --sort-by-file \
                 --keyword=_ --keyword=N_ --keyword=sgettext --keyword=vgettext --keyword=VGETTEXT \
                 --keyword=_n:1,2 --keyword=N_n:1,2 --keyword=sngettext:1,2 --keyword=vngettext:1,2 --keyword=VNGETTEXT:1,2 \
                 --files-from=%s --copyright-holder='Wesnoth development team' --msgid-bugs-address=https://bugs.wesnoth.org/ \
                 --output=$TARGET \
                 ; sed -i s/charset=CHARSET/charset=UTF-8/ $TARGET \
-                """ % (domain, join("po", domain, "POTFILES.in"))
+                """ % (domain, join("po", domain, "POTFILES_CPP.in"))
+                )
+
+        # py
+        with open(join(domain, "POTFILES_PY.in"), "w") as potfiles:
+            potfiles.writelines([line + "\n" for line in sorted(domain_tools_py[domain[7:]])])
+        tools_py = [Dir("#").File(x) for x in domain_tools_py[domain[7:]]]
+        if tools_py:
+            tools_py_pot = env.Command(
+                join(domain, domain + ".cpp.pot"),
+                sources_cpp,
```

Comment:
Should the last two lines be `domain + ".py.pot"` and `sources_py` ?

Reply:
> Should the last two lines be domain + ".py.pot" and sources_py ?

Yes, thanks for tracking this down. That explains your findings that ``wesnoth`` is emptied.

> renaming tools_py as sources_py

This also makes sense.